assets/js/blocks/price-filter/edit.js

```diff
@@ -151,7 +152,7 @@ export default function ( { attributes, setAttributes } ) {
 
 	return (
 		<>
-			{ getSetting( 'productCount', 0 ) === 0 ? (
+			{ blocksConfig.productCount === 0 ? (
```

Comment:
This is _slightly_ different than the previous implementation because `undefined !== 0` (before we defaulted to `0` if it was undefined). Practically, we probably wouldn't see this in a WordPress context but still worth noting I think.